docs/src/main/asciidoc/config.adoc

```diff
@@ -313,6 +313,42 @@ If the validation fails with the given configuration, the application will fail
 In the case of an interface being annotated with `@ConfigProperties`, the interface is allowed to extend other interfaces and methods from
 the entire interface hierarchy are used to bind properties.
 
+=== Using same ConfigProperties with different prefixes
+
+Quarkus also supports allows the use of the same `@ConfigProperties` object with different prefixes for each injection point using the `io.quarkus.arc.config.@ConfigPrefix` annotation.
+Say for example that `GreetingConfiguration` from above needs to be used for both the `greeting` prefix and the `other` prefix.
+In that case the code would look like so:
+
+`GreetingConfiguration.java`
+[source,java]
+----
+@ConfigProperties(prefix = "greeting")
+public class GreetingConfiguration {
+
+    @Size(min = 20)
+    public String message;
+    public String suffix = "!";
+
+}
+----
+
+`SomeBean.java`
+[source,java]
+----
+@ApplicationScoped
+public class SomeBean {
+
+    @Inject <1>
+    GreetingConfiguration greetingConfiguration;
+
+    @ConfigPrefix("other") <2>
+    GreetingConfiguration otherConfiguration;
+
+}
+----
+<1> At this injection point `greetingConfiguration` with use the `greeting` prefix since that is what has been defined on `@ConfigProperties`.
```

Comment:
```suggestion
<1> At this injection point `greetingConfiguration` will use the `greeting` prefix since that is what has been defined on `@ConfigProperties`.
```